src/SonataCacheBundle.php

```diff
@@ -26,14 +26,16 @@ public function build(ContainerBuilder $container)
 
     public function boot()
     {
-        $baseTemplateClass = $this->container->get('twig')->getBaseTemplateClass();
+        if ($this->container->has('twig')) {
+            $baseTemplateClass = $this->container->get('twig')->getBaseTemplateClass();
 
-        if (empty($baseTemplateClass)) {
-            return;
-        }
+            if (empty($baseTemplateClass)) {
+                return;
+            }
 
-        if (method_exists($baseTemplateClass, 'attachRecorder')) {
-            call_user_func([$baseTemplateClass, 'attachRecorder'], $this->container->get('sonata.cache.recorder'));
+            if (method_exists($baseTemplateClass, 'attachRecorder')) {
+                call_user_func([$baseTemplateClass, 'attachRecorder'], $this->container->get('sonata.cache.recorder'));
```

Comment:
Please linebreak lines over 120 chars (on my phone, can't check the actual length)

Reply:
Its exactly 120 chars :smile:

Reply:
Great then